src/apis/Keyboard.rei

```diff
@@ -18,20 +18,18 @@ module Easing: {
 };
 
 type screenRect = {
-  .
-  "screenX": float,
-  "screenY": float,
-  "width": float,
-  "height": float,
```

Comment:
Sorry for having missed this earlier, but could you update the .md file as well?

Reply:
Don't worry about it, I will update 🧑‍🏭